### PR TITLE
[ODS-4488] Fix broken composite features reported by the postman tests

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: Application/EdFi.Ods.Api.IntegrationTestHarness/Startup.cs [hunk 1]
```diff
@@ -13,7 +16,8 @@ public class Startup : OdsStartupBase
         public Startup(IWebHostEnvironment env, IConfiguration configuration)
             : base(env, configuration)
         {
-
+            AssemblyLoader.EnsureLoaded<Marker_EdFi_Ods_Composites_Test>();
+            AssemblyLoader.EnsureLoaded<Marker_EdFi_Ods_Profiles_Test>();
```
Comment (jamessmckay) [inline Application/EdFi.Ods.Api.IntegrationTestHarness/Startup.cs:20]: This change is not needed, we load all assemblies in the folder automatically now
```suggestion
```

File: Application/EdFi.Ods.Api.IntegrationTestHarness/Startup.cs [hunk 2]
```diff
@@ -3,6 +3,9 @@
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
 using EdFi.Ods.Api.Startup;
+using EdFi.Ods.Common.Extensions;
+using EdFi.Ods.Composites.Test;
+using EdFi.Ods.Profiles.Test;
```
Comment (jamessmckay) [inline Application/EdFi.Ods.Api.IntegrationTestHarness/Startup.cs:8]:
```suggestion
```

**Review by jamessmckay — CHANGES_REQUESTED** [review]

**Review by tbmeador-dlp — CHANGES_REQUESTED** [review]
@semalaiappan when you get a chance there is still some unaddressed comments on this PR.

**Review by tbmeador-dlp — CHANGES_REQUESTED** [review]
@semalaiappan do we need these remaining changes or can we delete this branch now?

**Comment by tbmeador-dlp** [comment]
reopening as these changes are required

**Review by tbmeador-dlp — APPROVED** [review]

**Pull request merged**